lib/packagekit-glib2/pk-client-helper.c

```diff
@@ -135,11 +135,13 @@ pk_client_helper_child_free (PkClientHelperChild *child)
 gboolean
 pk_client_helper_stop (PkClientHelper *client_helper, GError **error)
 {
-	PkClientHelperPrivate *priv = client_helper->priv;
+	PkClientHelperPrivate *priv = NULL;
```

Comment:
Would pk_client_helper_get_instance_private() work here? i.e. does it ignore NULL?

Reply:
Seems to be. Then we also don't really need ->priv anymore.

https://developer.gnome.org/gobject/stable/gobject-Type-Information.html says:
> It is safe to call _get_instance_private on NULL or invalid object since it's only adding an offset to the instance pointer. In that case the returned pointer must not be dereferenced.